### PR TITLE
Add support for account linking

### PR DESCRIPTION
**Pull request by kirre-bylund**

### CODE REVIEW

File: Runtime/Game/LootLockerSDKManager.cs [hunk 1]
```diff
@@ -7,6 +7,10 @@
 using LootLocker.LootLockerEnums;
 using static LootLocker.LootLockerConfig;
 using System.Linq;
+using static System.Net.WebRequestMethods;
+using static System.Runtime.CompilerServices.RuntimeHelpers;
+using System.Diagnostics;
+using File = System.IO.File;
```
Comment (JohannesLoot) [inline Runtime/Game/LootLockerSDKManager.cs:13]: Are these intentionally added?

File: Runtime/Game/LootLockerSDKManager.cs [hunk 2]
```diff
@@ -1026,6 +1030,90 @@ public static void WhiteLabelLoginAndStartSession(string email, string password,
...
+            LootLockerServerRequest.CallAPI(endpoint.endPoint, endpoint.httpMethod, onComplete: (serverResponse) => { LootLockerResponse.Deserialize(onComplete, serverResponse); });
+        }
+
+        /// <summary>
+        /// Check the status of an ongoing account linking process
+        /// https://ref.lootlocker.com/game-api/#check-account-link-status
+        /// </summary>
+        /// <param name="LinkID">The ID of the account linking process which was returned when starting the linking process</param>
+        /// <param name="onComplete">onComplete Action for handling the response of type LootLockerAccountLinkProcessStatusResponse</param>
+        public static void CheckStatusOfAccountLinkingProcess(string LinkID, Action<LootLockerAccountLinkProcessStatusResponse> onComplete)
```
Comment (JohannesLoot) [inline Runtime/Game/LootLockerSDKManager.cs:1062]: Same comment here as in the Unreal implementation with the naming

**Review by JohannesLoot — APPROVED** [review]
So far, so good!
Let me know if it needs a re-review when it has been tested and implemented "fully".

File: Runtime/Game/LootLockerSDKManager.cs (hunk 1 above)
Comment (kirre-bylund) [reply Runtime/Game/LootLockerSDKManager.cs:13]: Yeah, for some reason it didn't compile without it O.o

File: Runtime/Game/LootLockerSDKManager.cs [hunk 3]
```diff
@@ -1026,6 +1030,90 @@ public static void WhiteLabelLoginAndStartSession(string email, string password,
 
         #endregion
 
+        #region Account Linking
```
Comment (kirre-bylund) [inline Runtime/Game/LootLockerSDKManager.cs:1033]: Account linking is a working title, so don't merge until that's resolved.

File: Runtime/Game/LootLockerSDKManager.cs (hunk 2 above)
Comment (kirre-bylund) [reply Runtime/Game/LootLockerSDKManager.cs:1062]: Fixed

**Pull request merged**